.changeset/wise-carrots-float.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': minor
+---
+
+Removes the experimental `contentCollectionsCache` introduced in `3.5.0`.
```

Comment:
Just making sure we also have a note to add this to the list of experimental flags removed in the v5 upgrade guide!

Reply:
https://github.com/withastro/docs/pull/9507